ui/assets/css/_header.scss

```diff
@@ -22,3 +21,14 @@
     padding-left: 0.2rem;
   }
 }
+
+@media (max-width: 700px) {
```

Comment:
Will there be any issues defining the media queries in px? It looks like basscss defines breakpoints in ems. Should we [use](https://github.com/basscss/basscss-sass/blob/48212a612efaa913599bbd273724622a00e0a7f4/scss/_media-object.scss#L45) their variables?